third_party/mbedtls/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ set(OT_MBEDTLS_CONFIG_FILE "" CACHE STRING "The mbedTLS config file")
 set(ENABLE_TESTING OFF CACHE BOOL "Disable mbedtls test" FORCE)
 set(ENABLE_PROGRAMS OFF CACHE BOOL "Disable mbetls program" FORCE)
 
-add_subdirectory(repo)
+add_subdirectory(repo EXCLUDE_FROM_ALL)
```

Comment:
better use `EXCLUDE_FROM_ALL` on the whole `third_party` directory.

Reply:
agree.